doc/cleaning.rst

```diff
@@ -6,6 +6,44 @@ Cleaning
 
 .. currentmodule:: skrub
 
+:class:`Cleaner`: sanitizing a dataframe
+=========================================
+
+:class:`Cleaner` sanitizes a dataframe, transforming it to a more
+consistent data representation which is easier to work with: it detects
+null values represented as strings, parses dates, and removes
+uninformative columns (see docstring: :class:`Cleaner`).
```

Comment:
We display :class:`Cleaner` three times, should we remove the "see docstring" part?